src/main/java/org/prebid/server/spring/config/MetricsConfiguration.java

```diff
@@ -81,7 +83,16 @@ ScheduledReporter influxdbReporter(InfluxdbProperties influxdbProperties, Metric
                 influxdbProperties.getConnectTimeout(),
                 influxdbProperties.getReadTimeout(),
                 influxdbProperties.getPrefix());
-        final ScheduledReporter reporter = InfluxDbReporter.forRegistry(metricRegistry).build(influxDbSender);
+        Map<String, String> cfgTags = influxdbProperties.getTags();
```

Comment:
Hi, Michael!
Thank you for contribution in Prebid Server Java!

i've a couple of thoughts:
1. Please make `cfgTags` and `tags` variables to be final.
2. Use `Collections.emptyMap()` instead of `new LinkedHashMap<String, String>()`.

btw, you can use `ObjectUtils.defaultIfNull(...)` to get rid off ternary operator.

Reply:
Thank you!

BTW, it can be optimized as a one-line one variable:

```java
final Map<String, String> tags = ObjectUtils.defaultIfNull( 
    influxdbProperties.getTags(), 
    Collections.emptyMap()
);
``` 

Is it a good idea?

Reply:
Yes, great! Thanks -)